source/components/recipe-search/recipe-search.js

```diff
@@ -231,25 +231,209 @@ class RecipeSearch extends YummyRecipesComponent {
     let paramString = window.location.href.split("?")[1];
     let queryString = new URLSearchParams(paramString);
 
+    // Number # of recipes per page
+    const recipePerPage = 15;
+
     let paramArray = [];
     for (let p of queryString) {
       paramArray.push(p);
     }
 
     let searchRecipe = await database.searchByName(paramArray);
 
+    // Used by addEventListener to have access to this
+    let thisContainer = this;
+
     // Clear out recipe card grid before we append new cards
     this.shadowRoot.getElementById("recipe-card-grid").innerHTML = "";
 
-    for (const recipe of searchRecipe) {
-      /**
-       * Closes filters form when clicking on "X" icon.
-       */
-      this.shadowRoot
-        .getElementById("recipe-card-grid")
-        .appendChild(this.createRecipeCard(recipe.recipe, recipe.index));
+    // Apply Pagination
+    if (searchRecipe.length > recipePerPage) {
+      // Round up (recipe length)/(amount of recipes per page)
+      const pageCount = Math.ceil(searchRecipe.length / recipePerPage);
+      let pagDiv = this.shadowRoot.getElementById("recipe-pagination");
+
+      // Create left arrow icon for navigation
+      let leftArrow = document.createElement("a");
+      leftArrow.innerHTML = "&#8592;";
+      leftArrow.classList.add("pagination-arrow");
+
+      // If clicked on, click previous page anchor tag
+      leftArrow.addEventListener("click", () => {
+        let url = window.location.href;
+
+        let currPageNum = "";
+
+        let pageChunk = url.slice(url.indexOf("?page="), url.length);
+        let equalIndex = pageChunk.indexOf("=");
+
+        for (let i = equalIndex + 1; i < pageChunk.length; i++) {
+          currPageNum = currPageNum + pageChunk[i];
+        }
+
+        currPageNum = parseInt(currPageNum);
```

Comment:
Please specify base 10 for the `parseInt`. See below:
```suggestion
        currPageNum = parseInt(currPageNum, 10);
```